### PR TITLE
PTV-1786: Reduce arguments to WorkspaceAdministration

### PR DESCRIPTION
**Pull request by MrCreosote**
Minor cleanup in general

### CODE REVIEW

File: src/us/kbase/workspace/kbase/admin/WorkspaceAdministration.java [hunk 1]
```diff
@@ -67,9 +66,6 @@
  */
 public class WorkspaceAdministration {
 	
-	//TODO JAVADOC
-	//TODO TEST unit tests
```
Comment (MrCreosote) [inline src/us/kbase/workspace/kbase/admin/WorkspaceAdministration.java:71]: Turns out I wrote unit tests at some point but never deleted this TODO

**Comment by MrCreosote** [comment]
Try closing and reopening to get travis to pick this up

File: src/us/kbase/workspace/kbase/admin/AdminCommand.java [hunk 2]
```diff
@@ -26,7 +30,4 @@ public String getUser() {
 	public UObject getParams() {
 		return params;
 	}
-	
-	
-
```
Comment (ialarmedalien) [inline src/us/kbase/workspace/kbase/admin/AdminCommand.java:31]: I hope these weren't important!

File: src/us/kbase/workspace/kbase/admin/WorkspaceAdministration.java [hunk 3]
```diff
@@ -323,7 +330,8 @@ public Object runCommand(
 			final WorkspaceIdentity params = getParams(cmd, WorkspaceIdentity.class);
 			final WorkspaceIdentifier wsi = processWorkspaceIdentifier(
 							params.getWorkspace(), params.getId());
-			final WorkspaceInformation info = ws.getWorkspaceInformationAsAdmin(wsi);
+			final WorkspaceInformation info = wsmeth.getWorkspace().
```
Comment (ialarmedalien) [inline src/us/kbase/workspace/kbase/admin/WorkspaceAdministration.java:333]: sometimes GitHub's diff algorithm highlights the string that has been changed, sometimes it doesn't. It's nice to be kept on your toes.

**Review by ialarmedalien — APPROVED** [review]
👍

**Pull request merged**